docs/adr/README.md

```diff
@@ -0,0 +1,32 @@
+# Architectural Decision Log
+
+This log lists the architectural decisions for FxA.
+
+<!-- adrlog -- Regenerate the content by using "npx adr-log -i" from a shell in this directory. -->
```

Comment:
I honestly never ran this command when updating the TOC. I've just copy/pasted/updated link before it. Maybe we can remove the `<!-- adrlog` parts, @mozilla/fxa-devs objections? Do y'all use this command to create TOC?

Reply:
We *should* be running this command, but we forget. Alternatively, we might remove this and just give up on the TOC altogether if it's not useful (i.e. because we just use the github listing)

Reply:
Also I think `adr-log` will try to write `index.md` again anyway and won't follow this rename to `README.md`

Reply:
Let's just delete this file altogether since we don't use this command.

Reply:
Okay cool. I will delete this file!